runtime/testnet/src/lib.rs

```diff
@@ -462,13 +468,13 @@ impl pallet_staking::Config for Runtime {
 }
 
 parameter_types! {
-	pub const LaunchPeriod: BlockNumber = 28 * 24 * 60 * MINUTES;
-	pub const VotingPeriod: BlockNumber = 28 * 24 * 60 * MINUTES;
-	pub const FastTrackVotingPeriod: BlockNumber = 3 * 24 * 60 * MINUTES;
-	pub const MinimumDeposit: Balance = 100 * UNIT;
-	pub const EnactmentPeriod: BlockNumber = 30 * 24 * 60 * MINUTES;
-	pub const CooloffPeriod: BlockNumber = 28 * 24 * 60 * MINUTES;
-	pub const MaxProposals: u32 = 100;
+	pub const LaunchPeriod: BlockNumber = LAUNCH_PERIOD;
```

Comment:
I feel we should keep testnet configs separate from mainnet. It would be easier to test the node and then update configurations in mainnet once tested. wdyt ?

Reply:
I think it should be same, in case we find a bug we dont have to fix it in two places, for testing case I think we can use `#[(cfg="testing")]` I think.